### PR TITLE
Bump Sentry-Cocoa to 7.4.8

### PR DESCRIPTION
**Pull request by marandaneto**
## :pencil: Checklist

- [ ] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated the docs if needed
- [ ] All tests passing
- [ ] No breaking changes

### CODE REVIEW

**Comment by marandaneto** [comment]
cc @brustolin please review.

**Review by brustolin — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Unreleased
 
-* Bump: Sentry-Android to 5.2.4 and Sentry-Cocoa to 7.4.6 (#621)
+* Bump: Sentry-Android to 5.2.4 and Sentry-Cocoa to 7.4.8 (#621) (#627)
```
Comment (brustolin) [inline CHANGELOG.md:3]: l: Is this right? I never saw two PR references in the CHANGELOG.

Comment (marandaneto) [reply CHANGELOG.md:3]: it is, since #621 isn't published yet, no need to repeat again a new version bump

**Pull request merged**